src/Cdn/Cdn.Autorest/test/Test-AzFrontDoorCdnEndpointCustomDomain.Recording.json

```diff
@@ -116,7 +116,7 @@
         "Content-Type": [ "application/json; charset=utf-8" ],
         "Expires": [ "-1" ]
       },
-      "Content": "{\"customDomainValidated\":false,\"reason\":\"NotExist\",\"message\":\"Custom domain dose not exist in the profile. Please make sure the domain exsits in the profile.\"}",
+      "Content": "{\"customDomainValidated\":false,\"reason\":\"NotExist\",\"message\":\"Custom domain dose not exist in the profile. Please make sure the domain exists in the profile.\"}",
```

Comment:
No need to update recording files as they are generated and would be overwritten the next time when re-recorded.

Reply:
Reverted as requested